pkg/manifests/manifests.go

```diff
@@ -136,6 +137,13 @@ func (f *Factory) TunedDaemonSet() (*appsv1.DaemonSet, error) {
 	imageTuned := ntoconfig.NodeTunedImage()
 	ds.Spec.Template.Spec.Containers[0].Image = imageTuned
 
+	for i, e := range ds.Spec.Template.Spec.Containers[0].Env {
```

Comment:
Not necessarily an issue again, but highlights the inconsistency in variable names between `ds` here and `daemonset` above.